### ISSUE TITLE
Styling shawdow components does not work as expected and breaks on reload

### ISSUE THREAD

**Issue by mburger81** [ionic-team/stencil#879]
**Stencil version:**

```
@stencil/core@0.9.11
```

**I'm submitting a:**

[x] bug report
[ ] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://forum.ionicframework.com/ or https://stencil-worldwide.slack.com

**Current behavior:**

We have a battery component and a battery sass file, see this two files:
**battery.tsx**
```
@Component({
  tag: 'lan-battery',
  styleUrl: 'battery.scss',
  shadow: true
})
export class Battery {
```
**battery.scss**
```
:host(lan-battery) {
    display: flex;
    width: 100%;
    height: 100%;
```
If we load the dev server, anything is fine we can see the loaded style in shadow dom, see screenshot
![image](https://user-images.githubusercontent.com/3778892/41843956-17abb6c8-786f-11e8-99a5-ea23ebf03e54.png)
But if we change something in our scss file, for example add some empty lines and the component is reloading the style are not anymore there, see next screenshot
![image](https://user-images.githubusercontent.com/3778892/41844230-db824cec-786f-11e8-8d65-61d0299cbae4.png)

So there are two problems, first we are not sure why, but we have to apply the host to (lan-battery), should it not be enough doing like described in documentation (https://stenciljs.com/docs/styling)?
Not sure if this is a bug or we are misunderstanding something.

Bu at the end, for sure it is a bug that saving the file and so after reloading the component, many times the style is not loaded in shadow dom.

**Comment by mburger81** [comment]
The problem with the `:host(lan-battery)`, is not present. We can do `:host() {}` but this didn't work before, for the reason that the live reload seems to not work.

So the only problem we probably have is the live reload.

**Comment by manucorporat** [comment]
@mburger81 can you update to latest stencil `0.10.0-7` and see if it's fixed?

**Comment by mburger81** [comment]
@manucorporat 
I updated to `0.10.0.-9` which is the latest mentioned version in changelog, I removed dev server as described.

Now if I change some styles for example to the background color I can see the rebuild in DEV TOOLS (screen1) and SHELL CONSOLE, but it seems no reload of the component, or at least the color does not change.
If I then do a CRTL+F5 the page is reloaded without component shown, having a look to the dom the web component is present but there is no styles for shawdo dom (screen2).
If I stop and restart the server I got the changed color and the style is in dom (screen3)


**screen1**
![image](https://user-images.githubusercontent.com/3778892/42092880-cb63e792-7baa-11e8-81be-ca035722bc8d.png)
**screen2**
![image](https://user-images.githubusercontent.com/3778892/42093058-5935b8e8-7bab-11e8-858c-695d74c3e032.png)
**screen3**
![image](https://user-images.githubusercontent.com/3778892/42093127-9123517a-7bab-11e8-84a7-ae2d2cd70710.png)

**Comment by mburger81** [comment]
Sometimes there are another behavior, i change on value in my scss and click save, I see the dom element is reloaded but with the old values. I change color again and do save, now the dom is also reloaded but with the right values and color is changing. I change the color again and do save, dom refresh with old values, do another tentative with new value and now it works.

So:
change: not working
change: working
change: not working
change: working
... and so on


thx a lot

**Comment by mburger81** [comment]
Hello, this seems to be resolved in `"@stencil/core": "0.10.0-11"`, is this possible?

**Comment by adamdbradley** [comment]
Yes this may have been fixed up in the recent versions, thanks
